libs/shared/console-shared/src/lib/job-general-setting/job-general-setting.tsx

```diff
@@ -0,0 +1,122 @@
+import { BuildModeEnum } from 'qovery-typescript-axios'
+import { Controller, useFormContext } from 'react-hook-form'
+import { IconEnum, ServiceTypeEnum } from '@qovery/shared/enums'
+import { JobGeneralData, OrganizationEntity } from '@qovery/shared/interfaces'
+import { BlockContent, Icon, InputSelect, InputText } from '@qovery/shared/ui'
+import CreateGeneralGitApplication from '../create-general-git-application/create-general-git-application'
+import GeneralContainerSettings from '../general-container-settings/ui/general-container-settings'
+import EditGitRepositorySettingsFeature from '../git-repository-settings/edit-git-repository-settings-feature/edit-git-repository-settings-feature'
+
+export interface JobGeneralSettingProps {
+  organization?: OrganizationEntity
+  jobType: 'cron' | 'lifecycle'
```

Comment:
You need to use `ServiceTypeEnum`

Reply:
I read recently that it's better to make Union Type that Enum : https://www.bam.tech/article/should-you-use-enums-or-union-types-in-typescript

I am gonna refactor to have a Union Type here.